bootloader/common/pyi_launch.c

```diff
@@ -458,6 +458,7 @@ int pyi_launch_execute(ARCHIVE_STATUS *status, int argc, char *argv[])
      * If it's None(NULL), get the filesystem encoding by using direct
      * C calls and override it with correct value.
      */
+#ifndef WIN32
     if (!*PI_Py_FileSystemDefaultEncoding) {
```

Comment:
Why should FileSystemDefaultEncoding nor be handled in win32? Please add a comment about the reason directly to the code. Thanks.